test/helpers/balancer.js

```diff
@@ -0,0 +1,52 @@
+const LBPArtifact = require("../../imports/LiquidityBootstrappingPool.json");
+const LBPFactoryArtifact = require('../../imports/LiquidityBootstrappingPoolFactory.json');
+const VaultArtifact = require("../../imports/Vault.json");
+const AuthorizerArtifact = require("../../imports/Authorizer.json");
+const { constants } = require('@openzeppelin/test-helpers');
+const { ethers } = require("hardhat");
+
+const LBPFactory = async (setup) => {
```

Comment:
Btw I would root for consistent naming and following the [JS naming conventions](https://www.robinwieruch.de/javascript-naming-conventions). I need to remind myself to stick to this more conseuqnetly as well :) My suggestion would be to call:
- factories: `<contract_name>Factory`, here in line 31 for instance `vaultFactory`
- instantiated contracts: `<contract_name>Instance`, here in line 35 for instance `vaultInstance`
- functions that create contract instances: `get<Contract_name>Instance`

Reply:
naming suggestion for function: `getLbpFactoryInstance`

Reply:
@codebuster22  this is something for you too :D would be a good thing to remind each other on the naming conventions

Reply:
And thanks ofcourse Fabi